backend/src/main/java/pl/seahawks/vacationes/providers/ItakaProvider.java

```diff
@@ -0,0 +1,109 @@
+package pl.seahawks.vacationes.providers;
+
+import com.gargoylesoftware.htmlunit.BrowserVersion;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.*;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.context.annotation.Configuration;
+import pl.seahawks.vacationes.dto.OfferDto;
+import pl.seahawks.vacationes.dto.OfferRequirementsDto;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Configuration
+public class ItakaProvider implements Provider {
+
+  private final String BASE_URL_ITAKA = "https://www.itaka.pl";
```

Comment:
Taka luźna propozycja, moze by te URL przesnieść do propertisów i wstrzykiwać w konstruktorze, zaleta taka, ze przy zmianie url, mozna to podmienic bez zmiany kodu.